ddpui/core/pipelinefunctions.py

```
@@ -56,6 +57,38 @@ def setup_airbyte_sync_task_config(
+                    "OrgWarehouse for org=%s has no dbt_profile_secret_block — post-sync ops skipped",
+                    org_task.org.slug,
+                )
+            try:
+                client = WarehouseFactory.get_warehouse_client(warehouse)
+                for op in org_task.post_sync_transform.get("ops", []):
+                    if op.get("type") != "cast":
+                        continue
+                    sql = client.generate_cast_sql(op["schema"], op["table"], op["config"])
+                    if sql:
+                        post_sync_ops.append({"type": "cast", "sql": sql})
+            except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)